### PR TITLE
feat: DAH-2386: Change Email API

### PR DESCRIPTION
**Pull request by chadbrokaw**
[DAH-2386](https://sfgovdt.jira.com/browse/DAH-2386)

[DAH-2386]: https://sfgovdt.jira.com/browse/DAH-2386?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: app/javascript/util/accountUtil.tsx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+// We will strictly validate the email address using the following regex.
+// Devise will also validate the email address on the backend with a looser pattern.
+export const emailRegex = new RegExp(
```
Comment (tallulahkay) [inline app/javascript/util/accountUtil.tsx:3]: do we need this? is it used?

Comment (tallulahkay) [reply app/javascript/util/accountUtil.tsx:3]: jk I found it!! curious why it has to be diff from the one we can import from bloom?

**Comment by tallulahkay** [comment]
@chadbrokaw can you update the ticket? It's a little unclear because it says to handle banners in the description

File: app/javascript/util/accountUtil.tsx (hunk 1 above)
Comment (chadbrokaw) [reply app/javascript/util/accountUtil.tsx:3]: That RegEx was the one that is currently being used in AngularJS, definitely open to using the Bloom one tho! For more info on this decision, here is the Slack thread: https://exygy.slack.com/archives/C030EGQAKK2/p1722363802571229

**Comment by chadbrokaw** [comment]
> @chadbrokaw can you update the ticket? It's a little unclear because it says to handle banners in the description

Yes! Sorry that got mixed up. Non-error banners are coming in T's PR. I updated the ticket to reflect that

**Pull request merged**